### PR TITLE
Add title/slug search terms to filter description

### PR DESCRIPTION
**Pull request by theseanything**
# What
This adds the title or slug search terms to the filter description, removes full stop at end of description and formats the numbers using delimiters.

# Why
This helps user see what filters are currently applied. 

# Screenshots

## Before
image

## After
image

---
# Review Checklist
* [x] Changes in scope.
* [x] Added/updated unit tests.
* [x] Added/updated feature tests.
* [x] Added/updated relevant documentation.
* [x] Added to Trello card.

### CODE REVIEW

**Comment by theseanything** [comment]
[(2) Index page - Add search terms to filter description.](https://trello.com/c/AL1cBkpB/917-2-index-page-add-search-terms-to-filter-description)

File: app/views/content/_table_data_description.html.erb [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
+    <span class="table-header__param"><%= pagination.formatted_total_results %></span> results
+    <% if filter.search_terms? %>
+        for "<span class="table-header__param"><%= filter.search_terms %></span>"
+    <% end %>
     <% if filter.document_type? %>
-        for <span class="table-header__param"><%= filter.document_type %></span>
+        in <span class="table-header__param"><%= filter.document_type %></span>
     <% end %>
-    from <span class="table-header__param"><%= filter.organisation_name %></span>.
+    from <span class="table-header__param"><%= filter.organisation_name %></span>
```
Comment (erino) [inline app/views/content/_table_data_description.html.erb]: Was it intentional to remove the full stop?

File: app/presenters/filter_presenter.rb [hunk 2]
```diff
@@ -5,6 +5,14 @@ def initialize(search_parameters, document_types, organisations)
     @organisations = organisations
   end
 
+  def search_terms?
+    @search_parameters[:search_term].present?
```
Comment (erino) [inline app/presenters/filter_presenter.rb]: Could this call `#search_terms` rather then `@search_parameters[:search_term]`?

File: app/views/content/_table_data_description.html.erb (hunk 1 above)
Comment (theseanything) [reply app/views/content/_table_data_description.html.erb]: Yup

Comment (erino) [reply app/views/content/_table_data_description.html.erb]: Maybe I should have read the Trello card first

**Pull request merged**